sdcm/utils/raft/__init__.py

```diff
@@ -40,8 +46,32 @@ def clean_group0_garbage(self, raise_exception: bool = False) -> None:
     def diff_token_ring_group0_members(self) -> list[str]:
         ...
 
+    def get_log_patterns_for_decommission_abort(self) -> list[str]:
+        return self.decommission_begin_messages + self.decommission_end_messages
+
+    def get_message_waiting_timeout(self, message):
```

Comment:
I don't like this API which is based on strings which defined in too different places... 
it will go out of sync...